CHANGELOG.mdx

```diff
@@ -16,6 +16,8 @@ Prefix the change with one of these keywords:
 
 ## [Unreleased]
 
+- changed the location story to include the google api key as environment variable
```

Comment:
Entries should live under a heading, in this case ### Changed

Reply:
got it thanks !